packages/medusa-plugin-typesense/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "medusa-plugin-typesense",
+  "version": "0.1.0",
+  "description": "Typesense search plugin for Medusa",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/medusajs/medusa",
+    "directory": "packages/medusa-plugin-typesense"
+  },
+  "files": [
+    "dist"
+  ],
+  "author": "Medusa",
+  "license": "MIT",
+  "scripts": {
+    "prepare": "cross-env NODE_ENV=production yarn run build",
+    "test": "jest --passWithNoTests src",
+    "build": "tsc",
+    "watch": "tsc --watch"
+  },
+  "dependencies": {
+    "@medusajs/modules-sdk": "^1.8.8",
+    "@medusajs/utils": "^1.9.1",
+    "typesense": "^1.5.4"
```

Comment:
Could you use `1.7.0-1` of the `typesense` package?

I've added [some (long overdue) improvements](https://github.com/typesense/typesense-js/commit/7342a6f1becc48d2b37a93fba7b6e2ba318d92cf) to avoid having to create a separate client object for reads vs writes.